runtime/server/queries_metrics.go

```diff
@@ -224,3 +244,10 @@ func validateInlineMeasures(ms []*runtimev1.InlineMeasure) error {
 	}
 	return nil
 }
+
+func validateMeasures(ms []*runtimev1.InlineMeasure, measures []string) error {
+	if len(ms)+len(measures) == 0 {
+		return fmt.Errorf("no measures specified")
+	}
+	return nil
+}
```

Comment:
I think you can just use proto validations – only the `measures` slice defines what measures are returned, the inline measures I believe just inject extra definitions